lib/tasks/publishing_api.rake

```diff
@@ -23,4 +23,12 @@ namespace :publishing_api do
 
     HomepagePublisher.publish!(publishing_api, logger)
   end
+
+  desc "Send pages to the publishing-api"
+  task publish: [:environment] do
```

Comment:
should this perhaps be renamed to `publish_calendars` now it is in a less specialised app?